Mathlib/FieldTheory/Separable.lean

```diff
@@ -575,7 +575,7 @@ variable [Algebra K S] [Algebra K L]
 theorem AlgHom.card_of_powerBasis (pb : PowerBasis K S) (h_sep : (minpoly K pb.gen).Separable)
     (h_splits : (minpoly K pb.gen).Splits (algebraMap K L)) :
     @Fintype.card (S →ₐ[K] L) (PowerBasis.AlgHom.fintype pb) = pb.dim := by
-  let s := ((minpoly K pb.gen).map (algebraMap K L)).roots.toFinset
+  let s := ((minpoly K pb.gen).aroots L).toFinset
```

Comment:
This is also `rootSet`

Reply:
`Fintype.card_of_subtype` requires a `Finset`

Reply:
Ah but `s` can be inferred automatically.